xc/device.mli

```diff
@@ -80,7 +80,8 @@ sig
 	       -> mac:string -> carrier:bool 
 	       -> ?mtu:int -> ?rate:(int64 * int64) option
 	       -> ?protocol:protocol -> ?backend_domid:Xenctrl.domid 
-	       -> ?other_config:((string * string) list) 
+	       -> ?other_config:((string * string) list)
+	       -> ?static_ip_setting:((string * string) list)
```

Comment:
Yes, so we cannot use `extra_private_keys`, but we'll add a new argument `extra_xenserver_keys`, which is more generally usable than `static_ip_setting`.